cli/azd/pkg/output/ux/list_as_text.go

```diff
@@ -14,7 +14,7 @@ import (
 // ["foo", "bar"] 					=> "foo and bar"
 // ["foo", "bar", "axe"] 			=> "foo, bar and axe"
 // ["foo", "bar", ..., ..., "axe"] 	=> "foo, bar, ..., ... and axe"
-func ListAsText(items []string) string {
+func ListAsText(items []string, connectWord string) string {
```

Comment:
Can we have `AndListAsText()` and `OrListAsText()` ?   Both should call the same function where the `and` or `or` is passed and then add test cases for both.
Keep the implementation function private like `listAsText([]string, string)`.